features/prompt-registry.mdx

```diff
@@ -51,6 +51,14 @@ You can also optionally pass `version` to get an older version of a prompt. By d
 template_dict = promptlayer.prompts.get("my_template", version=3)
 ```
 
+### Model Parameters
+
+When you fetch a prompt template, you can get the model parameters that were used by passing in the `include_parameters=True` argument. This returns a tuple with the prompt template and the model parameters. For example:
```

Comment:
Rephrase the paragraph to (also make sure the last sentence is correct):


When fetching a prompt template, you can optionally retrieve model parameters by using `include_parameters=True`. This will provide you with a tuple containing both the prompt template and the model parameters. 

If no custom model parameters are set, the value will be an empty dictionary.

Reply:
GPT helped me clean up make the wording sound a little cleaner

Reply:
@jzone3 updated. Skipped last line because the dictionary will have attributes only with none values like
```py
{"model_provider": None, "model": None, "parameters": None}
```